.github/scripts/lib/pester-sharded-core.ps1

```
@@ -26,6 +26,181 @@ function Get-RealGitFiles {
+
+    $inherited = [Environment]::GetEnvironmentVariable((script:Get-RunDepthEnvName))
+    $depth = 0
+    if ($inherited -match '^\d+$') { $depth = [int]$inherited + 1 }
+
+    $displayPath = $TestsPath
+    $commit = $null
+    $treeState = 'unknown'
+    $notes = [System.Collections.Generic.List[string]]::new()
+
+    try {
+        try { $displayPath = [System.IO.Path]::GetFullPath($TestsPath) } catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)